### PR TITLE
Admin customer list page

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by light-di — APPROVED** [review]

File: src/main/java/RemoveNewCustomer.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    private String adminUsername;
+    private String adminPassword;
+
+    private Boolean isLoggedIn;
+
+    private String customerEmail;
+
+    public RemoveNewCustomer(Driver driver) {
+        this.driver = driver;
+
```
Comment (Tbotny) [inline src/main/java/RemoveNewCustomer.java:23]: delete empty line

File: src/main/java/pages/AdminCustomerListPage.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+public class AdminCustomerListPage extends AbstractPage {
+    private static final int CHECKBOX_INDEX = 0;
+    public enum TableTextValue
+    {
+        NAME_INDEX(1),
+        EMAIL_INDEX(2),
+        GROUP_INDEX(3),
+        IP_INDEX(4),
+        DATE_ADDED_INDEX(5)
+        ;
```
Comment (Tbotny) [inline src/main/java/pages/AdminCustomerListPage.java]: move to the line above

File: src/main/java/pages/AdminLoginPage.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+import driver.Driver;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AdminLoginPage extends AbstractPage {
+    @FindBy(xpath = "//div[@class = 'alert alert-danger alert-dismissible'][text() = ' No match for Username and/or Password.\n              ']")
+    private WebElement loginErrMsg;
+
```
Comment (Tbotny) [inline src/main/java/pages/AdminLoginPage.java:13]: delete empty line

File: src/main/java/pages/AdminLoginPage.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    @FindBy(xpath = "//input[@type = 'text' and @name = 'username' and @id = 'input-username']")
+    private WebElement usernameTextBox;
+    @FindBy(xpath = "//input[@type = 'password' and @name = 'password' and @id = 'input-password']")
+    private WebElement passwordTextBox;
+
```
Comment (Tbotny) [inline src/main/java/pages/AdminLoginPage.java:18]: delete empty line

File: src/main/java/pages/AdminLoginPage.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
+package pages;
+
+import driver.Driver;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AdminLoginPage extends AbstractPage {
+    @FindBy(xpath = "//div[@class = 'alert alert-danger alert-dismissible'][text() = ' No match for Username and/or Password.\n              ']")
+    private WebElement loginErrMsg;
```
Comment (Tbotny) [inline src/main/java/pages/AdminLoginPage.java]: Change all WebElement to HTML Element

**Review by Tbotny — APPROVED** [review]

**Review by TolkachDima — APPROVED** [review]

**Pull request merged**